crates/rollup-boost/src/tests/common/mod.rs

```diff
@@ -30,6 +29,7 @@ use std::collections::HashSet;
 use std::net::TcpListener;
 use std::path::PathBuf;
 use std::str::FromStr;
+use std::sync::Mutex;
```

Comment:
We should use tokio mutexes here

```suggestion
use tokio::sync::Mutex;
```